src/Service/Gateway.php

```diff
@@ -57,18 +67,19 @@ public function pay(
         RequestDataBag $dataBag,
         SalesChannelContext $salesChannelContext
     ): RedirectResponse {
+
         // Method that sends the return URL to the external gateway and gets a redirect URL back
         try {
             $pre_order = $this->processOrder($transaction,$salesChannelContext);
             $order = $this->ginger->createOrder($pre_order);
         } catch (\Exception $e) {
-            $this->helper->saveEMSLog($e->getMessage(), ['FILE' => __FILE__, 'FUNCTION' => __FUNCTION__, 'LINE' => __LINE__]);
-            print_r($e->getMessage()); exit;
+            throw new EmsPluginException($e->getMessage());
             throw new AsyncPaymentProcessException(
                 $transaction->getOrderTransaction()->getId(),
                 'An error occurred during the creating the EMS Online order' . PHP_EOL . $e->getMessage()
             );
```

Comment:
Remove this Exception we don't throwing it

Reply:
Done